### PR TITLE
Add a workflow to try and automatically fixup a PR

### PR DESCRIPTION
**Pull request by erikjohnston**
Hopefully useful for automatically fixing up a contributor's PR?

### CODE REVIEW

File: .github/workflows/fix_lint.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        continue-on-error: true
+        run: poetry run isort .
+
+      - name: Code style (black)
+        continue-on-error: true
+        run: poetry run black .
+
+      - name: Semantic checks (ruff)
+        continue-on-error: true
+        run: poetry run ruff .
```
Comment (clokep) [inline .github/workflows/fix_lint.yaml:42]: I think you want

```suggestion
        run: poetry run ruff --fix .
```

Comment (clokep) [reply .github/workflows/fix_lint.yaml]: Huh, OK! Maybe it didn't used to be. 🤷

**Review by clokep — APPROVED** [review]
Looks reasonable, I guess we won't really know until we test it?

**Comment by erikjohnston** [comment]
> Looks reasonable, I guess we won't really know until we test it?

Yeah, alas :(

**Pull request merged**